examples/blog-service/review_blog.bal

```diff
@@ -14,23 +14,26 @@
 // specific language governing permissions and limitations
 // under the License.
 
-public isolated function reviewBlog(Blog blog) returns Review|error => natural {
-    You are an expert content reviewer for a blog site that 
-        categorizes posts under the following categories: ${categories}
+import ballerina/ai;
 
-        Your tasks are:
-        1. Suggest a suitable category for the blog from exactly the specified categories. 
-           If there is no match, use null.
+public isolated function reviewBlog(Blog blog) returns Review|error => 
+    natural (check ai:getDefaultModelProvider()) {
```

Comment:
Shall we assign this to a new variable and use it, since this is an example, we needs to reflect what we are doing

Reply:
My bad, assumed this returned the single instance as discussed offline.